components/core/src/clp/regex_utils/regex_translation_utils.cpp

```diff
@@ -92,6 +93,14 @@ using StateTransitionFuncSig
  */
 [[nodiscard]] StateTransitionFuncSig dot_state_transition;
 
+/**
+ * Appends regex metacharacters literally to the wildcard string.
+ *
+ * These metacharacters are escaped by backslashes, so they have their special meanings suppressed.
+ * For metacharacters shared by the regex and the wildcard syntax, keep the escape backslashes.
```

Comment:
I'm not entirely sure what is meant here. Are you saying we keep the escape character and interpret it literally (I don't think so, but that is how it reads)?

It is also unclear what "these" refers to in this case.